kolibri/plugins/setup_wizard/assets/src/views/index.vue

```diff
@@ -154,7 +153,8 @@
         return !!this.username;
       },
       usernameValidityCheck() {
-        return /^\w+$/g.test(this.username);
+        const hasPuncRe = /[\s`~!@#$%^&*()-+={}\[\]\|\\\/:;"'<>,\.\?]/; // eslint-disable-line
```

Comment:
We should probably abstract the username validity functionality into core, using this, and then use it in all places that usernames are created or can be edited.
https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/management/assets/src/views/user-page/user-create-modal.vue#L120
https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/management/assets/src/views/user-page/user-edit-modal.vue#L22
https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/user/assets/src/views/profile-page/index.vue#L25

Reply:
Btw, I added username validation in all those places, with the exception of the user-edit-modal, which is up next, in this pr https://github.com/learningequality/kolibri/pull/2005